.gitignore

```diff
@@ -20,3 +20,4 @@ certs/
 acmedns*.json
 charts/
 src/
+Chart.lock
```

Comment:
Not sure if `Chart.lock` should be checked-in or not 🤔

I know that we do check-in `package-lock.json` and other similar lock files, but this might be an issue with optional dependencies that are disabled.. I will look into that for the future